css-cascade-6/Overview.bs

```diff
@@ -92,19 +92,23 @@ Importing Style Sheets: the ''@import'' rule</h2>
 
 <pre class='prod'>
 		@import [ <<url>> | <<string>> ]
-		        [ layer | layer(<<layer-name>>) ]?
-		        <<import-conditions>> ;
+		        [[ layer | layer(<<layer-name>>) ]
+		         || [ scope( [ (<<scope-start>>) ]? [ to (<<scope-end>>) ]? | <<scope-start>> ) ]
```

Comment:
```diff
- [ scope( [ (<<scope-start>>) ]? [ to (<<scope-end>>) ]? | <<scope-start>> ) ]
+ [ scope( [ [ (<<scope-start>>) ]? [ to (<<scope-end>>) ]? ]! | <<scope-start>> ) ]
```

(do not allow empty alternative)

Reply:
But `@import scope()` is valid, though. It gives you an "implicit scope", i.e. equivalent to `@scope { }`.

Reply:
Ok, that is intentional then. Sorry! =)

Reply:
Shouldn't this then be `@import scope` in parallel with `@import layer`?

I'd rather not have authors needing to remember:
| valid | invalid |
| --- | --- |
| `@import layer` | `@import layer()` |
| `@import scope()` | `@import scope` |

Reply:
@romainmenke Ah, yes, that's seems better.